packages/react-components/src/components/chart/legend/useChartsLegend.tsx

```diff
@@ -201,6 +227,20 @@ const useChartsLegend = ({
     });
     setColumnDefinitions([legendColumnDefinition, ...tcColumnDefinitions]);
 
+    const tcColumnDefinitionsTanstack = graphic.map((g) => {
```

Comment:
same with this... everything w.r.t tanstack should its own appropriate files

Reply:
updated